command.go

```diff
@@ -1311,6 +1311,30 @@ func (c *Command) AddGroup(groups ...*Group) {
 	c.commandgroups = append(c.commandgroups, groups...)
 }
 
+// RemoveGroup removes one or more command groups to this parent command.
+func (c *Command) RemoveGroup(groupIDs ...string) {
```

Comment:
i’m not sure, but I don’t think this will allow removing a group for the help or completion commands. Do we want to support removing groups for those two commands?

Reply:
Interesting use case. I guess in theory someone might want to remove those from a group that have been added. But they are also sort of the "default" commands you get. What do you think Marc?

Reply:
In my opinion, I would like cobra to give users as much freedom of development as possible. It is up to the developers to decide whether they want to keep the `help` and `completion` commands as existing groupies or remove them.

Reply:
I agree and I'm 100% for freedom of use. It more just becomes a matter of not breaking long standing existing features. But this is new functionality anyways. So I'm good with it!

Reply:
I fixed as follows. If the group id passed as argument match, the group id of the help and completion command is reset in the RemoveGroup method.  I would appreciate it if you check the code again.